src/overlays/actors/ovl_En_Horse/z_en_horse.c

```diff
@@ -1,19 +1,68 @@
 #include "z_en_horse.h"
+#include "overlays/actors/ovl_En_In/z_en_in.h"
 
 #define FLAGS 0x00000010
 
 #define THIS ((EnHorse*)thisx)
 
+typedef void (*EnHorseCsFunc)(EnHorse*, GlobalContext*, CsCmdActorAction*);
+typedef void (*EnHorseActionFunc)(EnHorse*, GlobalContext*);
+
+// z_horse.c
+void func_8006DD9C(Actor* actor, Vec3f* arg1, s16 arg2);
+
+void func_80028A54(GlobalContext* globalCtx, f32 arg1, Vec3f* vec);
+
 void EnHorse_Init(Actor* thisx, GlobalContext* globalCtx);
 void EnHorse_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void EnHorse_Update(Actor* thisx, GlobalContext* globalCtx);
 void EnHorse_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-extern UNK_TYPE D_02000230;
-extern UNK_TYPE D_02002AC0;
-extern UNK_TYPE D_06006530;
+void EnHorse_InitCutscene(EnHorse* this, GlobalContext* globalCtx);
+void EnHorse_InitHorsebackArchery(EnHorse* this);
+void EnHorse_InitFleePlayer(EnHorse* this);
+void EnHorse_ResetIdleAnimation(EnHorse* this);
+void EnHorse_StartIdleRidable(EnHorse* this);
+void EnHorse_InitInactive(EnHorse* this);
+void EnHorse_InitIngoHorse(EnHorse* this);
+
+void EnHorse_Frozen(EnHorse* this, GlobalContext* globalCtx);
+void EnHorse_Inactive(EnHorse* this, GlobalContext* globalCtx);
+void EnHorse_Idle(EnHorse* this, GlobalContext* globalCtx);
+void EnHorse_FollowPlayer(EnHorse* this, GlobalContext* globalCtx);
+void EnHorse_UpdateIngoRace(EnHorse* this, GlobalContext* globalCtx);
+void EnHorse_MountedIdle(EnHorse* this, GlobalContext* globalCtx);
+void EnHorse_MountedIdleWhinneying(EnHorse* this, GlobalContext* globalCtx);
+void EnHorse_MountedTurn(EnHorse* this, GlobalContext* globalCtx);
+void EnHorse_MountedWalk(EnHorse* this, GlobalContext* globalCtx);
+void EnHorse_MountedTrot(EnHorse* this, GlobalContext* globalCtx);
+void EnHorse_MountedGallop(EnHorse* this, GlobalContext* globalCtx);
+void EnHorse_MountedRearing(EnHorse* this, GlobalContext* globalCtx);
+void EnHorse_Stopping(EnHorse* this, GlobalContext* globalCtx);
+void EnHorse_Reverse(EnHorse* this, GlobalContext* globalCtx);
+void EnHorse_LowJump(EnHorse* this, GlobalContext* globalCtx);
+void EnHorse_HighJump(EnHorse* this, GlobalContext* globalCtx);
+void EnHorse_BridgeJump(EnHorse* this, GlobalContext* globalCtx);
+void EnHorse_CutsceneUpdate(EnHorse* this, GlobalContext* globalCtx);
+void EnHorse_UpdateHorsebackArchery(EnHorse* this, GlobalContext* globalCtx);
+void EnHorse_FleePlayer(EnHorse* this, GlobalContext* globalCtx);
+
+extern CutsceneData D_02000230[];
+extern CutsceneData D_02002AC0[];
+extern Gfx D_06006530[];
+
+AnimationHeader* sEponaAnimHeaders[] = { 0x06006D50, 0x06005584, 0x06004DEC, 0x06003CEC, 0x060075F0,
+                                         0x060032B0, 0x06001E2C, 0x06002470, 0x06002C38 };
+
+AnimationHeader* sHniAnimHeaders[] = { 0x06009FC4, 0x0600A6B4, 0x0600901C, 0x060085E0, 0x0600AF60,
+                                       0x06007B54, 0x0600506C, 0x06005684, 0x06005E20 };
+
```

Comment:
can you put trailing commas on the last entries and reformat, it will stop the awkward indentation from the formatter